load-tests/csv_processing/results/summary.csv

```diff
@@ -0,0 +1 @@
+
```

Comment:
need to add the headers

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/416/commits/b7e9cfc6d0c2173dbe7c9a8f91257a7d56cc5f44